static/styles/buttons.less

```diff
@@ -1,77 +1,78 @@
+@btnInsetShadow: inset 0px 2px 5px rgba(0,0,0,0.2);
+
 .btn {
-    margin: 0;
-    padding: 10px;
-    box-shadow: none;
-    background: white;
-    border: 1px solid @mediumGrey;
-    color: @mediumGrey;
-    display: inline-block;
+    display: block;
+    position: relative;
+    width: 100%;
+    padding: 15px;
+    padding-bottom: 12px;
+    font-family: 'Fira Sans', sans-serif;
     font-size: 14px;
-    text-decoration: none;
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
+    font-weight: 400;  
+    color: white;
+    line-height: 1;
+    border-radius: 4px;
+    border: medium none;
+    outline: medium none;
+    cursor: pointer;
```

Comment:
Probably not needed, but ok as it makes testing in-browser a little more natural.